### PR TITLE
fix/remove sitemap

### PR DESCRIPTION
**Pull request by leszczuu**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Combines:
https://github.com/BitBagCommerce/SyliusCmsPlugin/pull/321
with https://github.com/BitBagCommerce/SyliusCmsPlugin/pull/317
(completes sitemap plugin removal after https://github.com/BitBagCommerce/SyliusCmsPlugin/commit/22bb97d816e41759976de65d238e55b40e137d0e )
combines changes in docs, with moving sitemap plugin to optional dependencies
it should:
* describe how to install sitemap as an additional dependency and use sitemap providers from this bundle
* make sure all tests pass without sitemap plugin

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# UPGRADE FROM 2.4 TO 2.5
```
Comment (mikemix) [inline UPGRADE.md:1]: This "fix" introduces a BC break thus it would be an upgrade from 2.4 to 3.0

Comment (leszczuu) [reply UPGRADE.md:1]: Yes, anything when You make sitemap optional will introduce kind of BC break.
To continue using sitemap one would have to - require it through composer/ add to composer json, register bundle (in bundles.php or AppKernel), add back(import) sitemaps routing, add back(import) cms plugin's sitemaps providers.
Maybe it's good idea to consider this as a BC break, and make new version 3.0

**Comment by stefandoorn** [comment]
Great :-) Hope this will be included soon, it seems a lot of users don't want to have it installed by default.

**Comment by stefandoorn** [comment]
@patrick477 Could this be included in the next release?

**Comment by patrick477** [comment]
yes, it will be in v3.0

**Comment by stefandoorn** [comment]
The first beta of v3 still includes it, @patrick477 

In the meanwhile I've also released `v2.0.0-alpha.1` of the Sitemap Plugin which requires some changes because of BC. Will you adjust the providers or do you want a PR? We can also think about splitting the providers into a separate repository/package that both requires the CMS Plugin and the Sitemap Plugin and combines best of both. WDYT?

**Comment by BartoszWojdalowicz** [comment]
@leszczuu you should also remove import sitemap from config.yml otherwise it throw error when install from 3.0 branch. 

" Bundle "SitemapPlugin" does not exist or it is not enabled. Maybe you forgot to add it in the "registerBundles()" method of your "App\Kernel.php" file? in @SitemapPlugin/Resources/config/config.yaml (which is being imported from "SyliusCmsPlugin/src/Resources/config/config.yml"). Make sure the "SitemapPlugin" bundle is correctly registered and loaded in the application kernel class. If the bundle is registered, make sure the bundle path "@SitemapPlugin/Resources/config/config.yaml" is not empty.

**Pull request merged**